python-checks/src/test/resources/checks/checkUtils/isSelfTest.py

```diff
@@ -0,0 +1,12 @@
+class C:
+    def returnsSelf01(self):
```

Comment:
For completeness, you could probably add a test with `@staticmethod` and `@classmethod` decorators.
In those cases the first parameter wouldn't be "self"